database/pgsql/testutil.go

```diff
@@ -1,262 +1,194 @@
-// Copyright 2018 clair authors
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
 package pgsql
 
```

Comment:
This is another thing I don't like about this PR. 
We need the pgsql implementation to implement the interfaces only used in the tests. 
While it makes sense, I don't know if there are better solutions.